samples/sdcrepo-secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sdcrepo-secret
+  namespace: powerflex
```

Comment:
Are we hardcoding this NS? Even if we are hardcoding any specific reason for `powerflex` instead of `vxflexos`. For any chance the init container can only access the secret in `powerflex` NS?